packages/google_maps_routes_api/lib/src/types/index.dart

```diff
@@ -0,0 +1,12 @@
+export 'package:google_maps_routes_api/src/types/enums.dart';
```

Comment:
Licence missing. I guess it is needed in every file. After deleting that analysis file the IDE probably will tell you where it is missing.